diesel/src/mysql/bind_collector.rs

```diff
@@ -1,15 +1,15 @@
-use super::{Mysql, MysqlType};
+use super::{Mysql, MysqlTypeMetadata};
 use query_builder::BindCollector;
 use result::Error::SerializationError;
 use result::QueryResult;
 use serialize::{IsNull, Output, ToSql};
-use sql_types::{HasSqlType, IsSigned};
+use sql_types::HasSqlType;
 
 #[derive(Default)]
 #[doc(hidden)]
 #[allow(missing_debug_implementations)]
 pub struct MysqlBindCollector {
-    pub(crate) binds: Vec<(MysqlType, IsSigned, Option<Vec<u8>>)>,
```

Comment:
Any reason why we need to keep the custom bind collector? Wouldn't it be easier to just use `RawBytesBindCollector` here?